internal/handlers/common/error.go

```diff
@@ -34,20 +34,46 @@ const (
 	// For ProtocolError only.
 	errInternalError = ErrorCode(1) // InternalError
 
-	ErrBadValue          = ErrorCode(2)     // BadValue
-	ErrFailedToParse     = ErrorCode(9)     // FailedToParse
-	ErrTypeMismatch      = ErrorCode(14)    // TypeMismatch
-	ErrNamespaceNotFound = ErrorCode(26)    // NamespaceNotFound
-	ErrNamespaceExists   = ErrorCode(48)    // NamespaceExists
-	ErrCommandNotFound   = ErrorCode(59)    // CommandNotFound
-	ErrInvalidNamespace  = ErrorCode(73)    // InvalidNamespace
-	ErrNotImplemented    = ErrorCode(238)   // NotImplemented
-	ErrSortBadValue      = ErrorCode(15974) // Location15974
-	ErrInvalidArg        = ErrorCode(28667) // Location28667
-	ErrSliceFirstArg     = ErrorCode(28724) // Location28724
-	ErrProjectionInEx    = ErrorCode(31253) // Location31253
-	ErrProjectionExIn    = ErrorCode(31254) // Location31254
-	ErrRegexOptions      = ErrorCode(51075) // Location51075
+	// ErrBadValue for wrong input.
+	ErrBadValue = ErrorCode(2) // BadValue
+
+	// ErrFailedToParse failed to parse user input.
+	ErrFailedToParse = ErrorCode(9) // FailedToParse
```

Comment:
Overall, here and below I miss verbs in comments. Usually, in the Go source code, there is a verb explaining what the variable is used for. For example, for errors, it could be "indicates" [like in the os package](https://cs.opensource.google/go/go/+/refs/tags/go1.18.2:src/os/error.go;l=18).

On the other hand, in the Go source code, not all the errors are documented as detailed as you did it here :) So, please treat my comment as an opinion, not a call for action.

Reply:
I welcome your feedback. I believe you added an example here to be more understandable to read when I add this. Anyway, this is a pure PR for comments.